web3/eth.py

```diff
@@ -458,7 +458,7 @@ async def call(
         return await self._call(transaction, block_identifier, state_override)
 
 
-class Eth(BaseEth, Module):
```

Comment:
`BaseEth` already inherits from `Module` so this didn't seem necessary.

Reply:
Nice catch!